src/main/java/at/hannibal2/skyhanni/config/storage/ProfileSpecificStorage.kt

```diff
@@ -192,7 +192,7 @@ class ProfileSpecificStorage {
 
         class HitmanStatsStorage {
             @Expose
-            var availableEggs: Int? = null
+            var availableHitmanEggs: Int = 0
```

Comment:
cant the value be null for legacy users and then cause a config problem without migration fix?

Reply:
the key changed, if it was null before, it won't map to the new name